### ISSUE TITLE
Add real plaintext mode

### ISSUE THREAD

**Issue by vitoreiji** [tutao/tutanota#1588]
Currently all emails are composed as HTML, the Plaintext user setting does:

* hide the toolbar
* ~~remove all formatting when saving the draft / sending the email~~
* for external recipients, generate a plain text version from the HTML and send the mime type text/plain

It would be nice to be able to compose emails in true plaintext and send for internal users as well.

**Comment by vitoreiji** [comment]
Related: #1583 #1559

**Comment by bedhub** [comment]
We should find a solution to handle blockquotes #1599

**Comment by bfrog** [comment]
Yes please, I would love to see real plaintext email editing not mangle quoted text!

**Comment by temporaryrespite** [comment]
> Currently all emails are composed as HTML, the Plaintext user setting does:
>  * remove all formatting when saving the draft / sending the email

It should say: "DOES NOT remove all formatting when saving the draft / sending the email"

I can copy/paste text from webpages and the formatting remained and can still be seen in the email in the `Sent` folder.

Tutanota v3.66.5

**Comment by vitoreiji** [comment]
@temporaryrespite you're right, we changed this behavior in a recent change.

**Comment by johnbotris** [comment]
We should revisit this. Plaintext mode should include stripping formatting when pasting, without having to type Ctrl+Shift+V

**Comment by t-900-a** [comment]
Many reasons to include a plaintext mode.
https://useplaintext.email/#why-plaintext

**Comment by gmejzej** [comment]
Is this bug about giving users a Plain Text viewing option? If so, can this PLEASE be moved to the top of the list over everything else? Plain Text viewing has been outstanding for years. This is a lot more important than Conversation View IMHO.

**Comment by SuperJugy** [comment]
I have a question but not sure if I should make a separate Issue for this. This is my use case:

I prefer having HTML formatting by default for my emails, but there are some mailing lists (Linux mailing lists in particular) that need any emails sent to them to be in plain/text; which require me to go and change the global setting to plain/text each time I need to send an email to those addresses and then turn it back to HTML.

I would prefer to have a "per email" option to send that specific email as plain text.

**Comment by ballo** [comment]
May mailing lists ban html. I need to find a different provider

**Comment by MasterInQuestion** [comment]
Currently "Plain Text" emails get poorly transformed into HTML.
    Some communities probably in particular distaste such false Plain Text...
    .
https://www.gnu.org/software/coreutils/#maintainers
    “Mailing List Etiquette:
    Please do not send messages encoded as HTML nor encoded as base64 MIME nor included as multiple formats. Please send messages as plain text.”
\
\
    "We will need to take care of not accidentally interpreting them as HTML"
    I've done thorough research with various markups... due to the invariable wrestling.

    For proper presenting in HTML, could be just alike:
    https://github.com/MasterInQuestion/Markup/blob/main/AAA.htm
    ; only "`<`", "`&`" need escaping.
    (to "`&lt;`", "`&amp;`" respectively)
[ ^
    Note "`<`" doesn't need escaping, if the surrounding couldn't constitute valid XML tag.
    I.e. escape only /<a href="https://regex101.com/?flavor=javascript&regex=%3C(?=%5Ba-z%5C/!?%5D)&flags=i">`<(?=[a-z\/!?])`</a>/i
    "`&`" needs escaping only when the following would make it an [XML/HTML entity](https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references).
    (but the exact logic is tricky as the buggy "`&lt`" alike might work) ]

    Keep in mind that over-escaping is horrendous practice:
    Effectively be code obfuscation...

**Comment by charlag** [comment]
@MasterInQuestion I'm not sure I understand what you are asking/suggesting

**Comment by MasterInQuestion** [comment]
You wrote:
[ https://github.com/tutao/tutanota/issues/1588#issuecomment-907051952](https://github.com/tutao/tutanota/issues/1588#issuecomment-907051952)
    "We will need to take care of not accidentally interpreting them as HTML"

    And I presented the caveats, and potential solution.
